src/service_main.rs

```diff
@@ -24,20 +24,33 @@ extern crate sendgrid;
 extern crate serde;
 #[macro_use]
 extern crate serde_derive;
+extern crate failure;
+extern crate mozsvc_common;
 extern crate serde_json;
+extern crate slog_async;
+extern crate slog_mozlog_json;
+extern crate slog_term;
+#[macro_use(
+    slog_o, slog_info, slog_kv, slog_log, slog_record, slog_b, slog_record_static, slog_error
+)]
+extern crate slog;
 
 mod app_errors;
 mod auth_db;
 mod bounces;
 mod deserialize;
 mod duration;
+mod logging;
 mod providers;
 mod send;
 mod settings;
 mod validate;
 
 fn main() {
+    let config = settings::Settings::new().expect("Config error.");
```

Comment:
Seems wrong that we're doing this before the inject-settings change, but I guess it's fine if we know that one's coming soon? :man_shrugging:

Reply:
Yeah, I only realized that after the PR was already sent. I guess we can also wait for that one before merging this one. When the inject-settings is accepted I rebase this one, etc.